utils/enums.py

```diff
@@ -99,14 +99,38 @@ def build_enum_data(enum_class):
     from enum import IntFlag
 
     class HtkIntFlag(IntFlag):
-        """ HTK IntFlag
+        """HTK IntFlag
 
         Wrapper around Python's built-in `enum.IntFlag` to provide extra
         functionalities.
 
         Requires: Python 3.6 or greater
         Reference: https://docs.python.org/3.6/library/enum.html#enum.IntFlag
         """
+
+        @classmethod
+        def display(cls, int_value):
+            """Get human-readable display names for combined flag values.
+
+            Decomposes combined flag values using list_flags(), then uses
+            get_enum_symbolic_name() to format each flag.
+
+            Args:
+                int_value: Combined flag value (e.g., 3 = BICEPS_BRACHII | BRACHIALIS)
+
+            Returns:
+                List of human-readable flag names
+
+            Example:
+                Muscle.display(3)  # ['Biceps Brachii', 'Brachialis']
```

Comment:
If you're going to include this example, please include the sample class implementation with values of `0, 1, 2, 4, 8` filled in.